.github/workflows/build-test-macos-plot_sync.yml

```diff
@@ -65,13 +65,25 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-pip-
 
-    - name: Checkout test blocks and plots
-      uses: actions/checkout@v3
+    - name: Daily (POC) cache key invalidation for test blocks and plots
```

Comment:
@AmineKhaldi Your new code was not being included here.

Reply:
This was handled in https://github.com/Chia-Network/chia-blockchain/pull/11092.